syz-verifier/execresult_test.go

```diff
@@ -136,7 +136,8 @@ func TestCompareResults(t *testing.T) {
 			},
 		}}
 
-	for _, test := range tests {
+	for _, curTest := range tests {
```

Comment:
It's better to not introduce 2 different names:
https://go.dev/doc/faq#closures_and_goroutines
Currently it's possible to use curTest by mistake in the closure.

Reply:
Thanks, done.